notebooks/QuantumChromoDynamicsExample.ipynb

```diff
@@ -0,0 +1,419 @@
                                                                                                                                      │                                 │                │                              │                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         │                     │        │                  │                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 │                                            │        ││                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
```

Comment:
It would be nice to use an EstimateMetadata object to keep things consistent with the other notebooks and to ease integration with the Metadata PR. Also would be good practice to use explicit argument assignments and not positional assignments. ex func(x=x, y=y) rather than func(x,y)